<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-N3S3FJH" failures="0" tests="2" name="valuesite.tests.SucursalTestRT" time="0.034" errors="1" timestamp="2024-05-06T09:35:54 CLT" skipped="1">
  <testcase classname="valuesite.tests.SucursalTestRT" name="nombreObligatorio" time="0.000">
    <skipped/>
  </testcase> <!-- nombreObligatorio -->
  <system-out/>
  <testcase classname="valuesite.tests.SucursalTestRT" name="setUp" time="0.034">
    <error message="Driver must be set" type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException: Driver must be set
at org.openqa.selenium.internal.Require.nonNull(Require.java:60)
at org.openqa.selenium.interactions.Actions.<init>(Actions.java:59)
at valuesite.componentesreusables.ActionsHelper.<init>(ActionsHelper.java:17)
at valuesite.pageobjects.SucursalRT.<init>(SucursalRT.java:27)
at valuesite.tests.SucursalTestRT.setUp(SucursalTestRT.java:20)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- setUp -->
  <system-out/>
</testsuite> <!-- valuesite.tests.SucursalTestRT -->
